src/Google/Ads/GoogleAds/Util/V8/GoogleAdsErrors.php

```diff
@@ -111,7 +111,7 @@ public static function fromFailure(
                 $index = $element->getIndex();
                 if (
                     ($fieldName === "operations" || $fieldName === "mutate_operations"
-                    || $fieldName === "conversions") && $index == $operationIndex
+                    || $fieldName === "conversions" || $fieldName === "conversion_adjustments") && $index == $operationIndex
```

Comment:
@PierrickVoulet is it ok if we cast fieldName as array of fieldName and we can check in array with clean code ?

Reply:
Not sure I understand correctly your suggestion, could you provide a concrete example or [a GitHub Gist](https://gist.github.com/)?

Reply:
I mean we can do something like

```
private $supportedFieldNames = ['conversions','conversion_adjustments','operations'];

In line 114 

if(in_array($fieldName, $this->supportedFieldNames) && $index === $operationIndex) {

}

```

Reply:
Makes sense, the best would be to have it as a constant though.

Reply:
@PierrickVoulet thanks, i have updated it and when will this be merged and release , since i am implementing custom error handling solutions :)